react/test/reactTextMask.spec.js

```diff
@@ -312,6 +312,47 @@ describe('MaskedInput', () => {
     maskedInput.onChange()
     expect(maskedInput.textMaskInputElement.update.callCount).to.equal(1)
   })
+
+  // test fix for issues #230, #483, #778 etc.
+  it('work correct in stateful Component', () => {
+    class StatefulComponent extends React.Component {
+      constructor(...args) {
+        super(...args)
+
+        this.state = {value: '1234'}
+        this.onChange = this.onChange.bind(this)
+      }
+
+      onChange(e) {
+        this.setState({value: e.target.value})
```

Comment:
Can you also apply a change in the `mask` field to prove the point that your fix only solves it for the `.value` ?

Reply:
Ahh it was merged already, ok...